tap_mixpanel/schema.py

```diff
@@ -132,17 +163,18 @@ def get_schemas(client, properties_flag):
             schema=schema,
             key_properties=stream_metadata.key_properties,
             valid_replication_keys=stream_metadata.replication_keys,
-            replication_method=stream_metadata.replication_method
+            replication_method=stream_metadata.replication_method,
         )
 
         mdata = metadata.to_map(mdata)
 
         if stream_metadata.replication_keys:
-                mdata = metadata.write(
-                    mdata,
-                    ('properties', stream_metadata.replication_keys[0]),
-                    'inclusion',
-                    'automatic')
+            mdata = metadata.write(
+                mdata,
+                ("properties", stream_metadata.replication_keys[0]),
+                "inclusion",
+                "automatic",
+            )
```

Comment:
```suggestion
            mdata = metadata.write(
                mdata,
                ("properties", stream_metadata.replication_keys[0]),
                 "inclusion",
                 "automatic",
            )
```

Reply:
I think it's the right way to do it. upon adding an extra space before `inclusion` and `automatic` PEP throws this warning `PEP 8: E131 continuation line unaligned for hanging indent`